vms/event/templates/event/edit.html

```diff
@@ -1,213 +1,213 @@
+                        <label class="col-md-2 control-label">{% trans "Address" %}</label>
+                        <div class="col-md-8">
+                            <input class="form-control" placeholder="{% blocktrans %}Event Address{% endblocktrans %}" type="text" name="address" value="{% if form.address.value %}{{ form.address.value }}{% endif %}">
+                            <p class="help-block">
+                                <strong>
+                                    {% for error in form.address.errors %}
+                                        {{ error }}
+                                    {% endfor %}
+                                </strong>
+                            </p>
+                        </div>
+                    </div>
+                {% else %}
+                    <div class="form-group">
+                        <label class="col-md-2 control-label">{% trans "Address" %}</label>
+                        <div class="col-md-8">
+                            <input class="form-control" placeholder="{% blocktrans %}Event Address{% endblocktrans %}" type="text" name="address" value="{% if form.address.value %}{{ form.address.value }}{% endif %}">
+                        </div>
+                    </div>
+                {% endif %}
+
+				{% if form.venue.errors %}
+                    <div class="form-group has-error">
+                        <label class="col-md-2 control-label">{% trans "Venue" %}</label>
+                        <div class="col-md-8">
+                            <input class="form-control" placeholder="{% blocktrans %}Event Venue{% endblocktrans %}" type="text" name="venue" value="{% if form.venue.value %}{{ form.venue.value }}{% endif %}">
+                            <p class="help-block">
+                                <strong>
+                                    {% for error in form.venue.errors %}
+                                        {{ error }}
+                                    {% endfor %}
+                                </strong>
+                            </p>
+                        </div>
+                    </div>
+                {% else %}
+                    <div class="form-group">
+                        <label class="col-md-2 control-label">{% trans "Venue" %}</label>
+                        <div class="col-md-8">
+                            <input class="form-control" placeholder="{% blocktrans %}Event Venue{% endblocktrans %}" type="text" name="venue" value="{% if form.venue.value %}{{ form.venue.value }}{% endif %}">
+                        </div>
+                    </div>
+                {% endif %}
+                <div class="form-group">
+                    <div class="col-md-12 col-md-offset-2">
+                        <button class="btn btn-primary" type="submit">{% trans "Save" %}</button>
+                    </div>
+                </div>
+            </fieldset>
+        </form>
```

Comment:
@tapasweni-pathak No changes are made in this file. May be it came due to the testing I have done locally. Sorry.